contracts/mock/MockERC20WithFee1.sol

```diff
@@ -0,0 +1,67 @@
+pragma solidity ^0.6.0;
+
+import "@openzeppelin/contracts/math/SafeMath.sol";
+
+import "../libraries/ERC20.sol";
+import "../interfaces/IERC20Extended.sol";
+
+contract MockERC20WithFee1 is ERC20, IERC20Extended {
+    using SafeMath for uint256;
+
+    constructor(
+        string memory _name,
+        string memory _symbol,
+        uint256 _totalSupply
+    ) public ERC20(_name, _symbol) {
+        _mint(msg.sender, _totalSupply);
+    }
+
+    function _transfer(
+        address from,
+        address to,
+        uint256 value
+    ) internal override {
+        uint256 transferAmount = (value * 99) / 100;
+        uint256 burnAmount = value.sub(transferAmount);
```

Comment:
burn here is the fee?

Reply:
normally, we will create an address to receive the fee. But in this context, address to collect fee is unimportant so we can send fee to 0x00.

For user, it creates the same experience